packages/clay-multi-select/src/index.tsx

```diff
@@ -357,7 +357,7 @@ const ClayMultiSelect = React.forwardRef<HTMLDivElement, IProps>(
 						!disabledClearAll &&
 						(inputValue || items.length > 0) && (
 							<ClayInput.GroupItem shrink>
-								<button
+								<ClayButtonWithIcon
```

Comment:
@kresimir-coko don't forget to pass a `displayType`, by using the default "primary" it's not going to look like we want it to.

Reply:
Good catch @julien, I'll take a look at that soon

Reply:
:+1: